banyan-macos-v3

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
```

Comment:
Add: `set -euo pipefail`

bash 'strict' mode. Makes the script exit if there is an error.